SDWebImage/Core/SDImageGraphics.h

```diff
@@ -19,9 +19,9 @@
 /// Returns the current graphics context.
 FOUNDATION_EXPORT CGContextRef __nullable SDGraphicsGetCurrentContext(void) CF_RETURNS_NOT_RETAINED;
 /// Creates a bitmap-based graphics context and makes it the current context.
-FOUNDATION_EXPORT void SDGraphicsBeginImageContext(CGSize size);
+FOUNDATION_EXPORT void SDGraphicsBeginImageContext(CGSize size) NS_SWIFT_NAME(SDGraphicsBeginImageContext(size:));
```

Comment:
Don't use Any C API for Swift....

Actually, I suggest to put these into class `SDImageCoderHelper` or even remove this.

Because we already provide a helper for `SDGraphicsImageRenderer`...

To be checked later.

Reply:
Got it.